auth-jwt-service/src/main/java/io/stargate/auth/jwt/AuthzJwtService.java

```diff
@@ -170,9 +171,8 @@ private void preCheckDataReadWrite(JSONObject stargateClaims, List<TypedKeyValue
       // If one of the columns exist as a field in the JWT claims and the values do not match then
       // the request is not allowed.
       if (stargateClaims.has(STARGATE_PREFIX + typedKeyValue.getName())) {
```

Comment:
This is probably outside the scope of this PR, but since we discussed it... Would it make sense to deny the request if the claims contain a key that is not included in the provided list of `TypedKeyValue`? 

As discussed this may be the case in range deletes.

@dougwettlaufer : WDYT?

Reply:
Also, for SELECTs with `ALLOW FILTERING` I guess we may not always get the full set of PK values.

Reply:
Right, I'll need to look into adding that.